projects/element-ng/chat-messages/si-action-bar.component.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { CdkMenuTrigger } from '@angular/cdk/menu';
+import { Component, inject, input } from '@angular/core';
+import { SiIconComponent } from '@siemens/element-ng/icon';
+import {
+  MenuItemAction,
+  SiMenuActionService,
+  SiMenuFactoryComponent
+} from '@siemens/element-ng/menu';
+import { SiTranslatePipe, t, TranslatableString } from '@siemens/element-translate-ng/translate';
+
+export interface ActionBarItem {
+  /** ID that will be attached to the DOM node. */
+  id?: string;
+  /** Label that is shown to the user. */
+  label: TranslatableString;
+  /**
+   * Icon used to represent the action
+   */
+  icon: string;
+  /**
+   * Action that called when the item is triggered.
+   * A function will be called, a string will be dispatched to the {@link SiMenuActionService}.
+   */
+  action: ((actionParam: any, source: this) => void) | string;
+  /** Whether the menu item id disabled. */
+  disabled?: boolean;
+}
+
+/** @experimental */
+@Component({
+  selector: 'si-action-bar',
+  imports: [SiIconComponent, SiMenuFactoryComponent, CdkMenuTrigger, SiTranslatePipe],
+  templateUrl: './si-action-bar.component.html',
+  host: {
+    class: 'd-flex gap-4'
+  }
+})
+export class SiActionBarComponent {
+  private menuActionService = inject(SiMenuActionService, { optional: true });
+
+  /**
+   * Primary actions available for this message (thumbs up/down, copy, retry, etc.)
+   * All actions displayed inline
+   * @defaultValue []
+   */
+  readonly actions = input<ActionBarItem[]>([]);
+
+  /**
+   * Actions available in dropdown menu
+   * @defaultValue []
+   */
+  readonly dropdownActions = input<MenuItemAction[]>([]);
```

Comment:
Should this be a `MenuItem`, as we also want to support separators and all other menu item variants.

Reply:
Done.